### PR TITLE
codewhisperer: enable crossfile for all Java users

### PR DESCRIPTION
**Pull request by Will-ShaoHua**
## Solution
* Cancel cross file A/B testing (expand to ALL java users)

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/codewhisperer/util/userGroupUtil.ts [hunk 1]
```diff
@@ -51,10 +51,7 @@ export class CodeWhispererUserGroupSettings {
     }
 
     private guessUserGroup(): UserGroup {
```
Comment (leigaol) [inline src/codewhisperer/util/userGroupUtil.ts:53]: help me understand why this enabled crossfile for java users?

Comment (Will-ShaoHua) [reply src/codewhisperer/util/userGroupUtil.ts:53]: The userGroup is just the treatment group with feature name on it. So here I remove all usages of crossfile group.

File: src/codewhisperer/util/editorContext.ts [hunk 2]
```diff
@@ -94,10 +93,9 @@ export async function buildListRecommendationRequest(
     // (1) User is CrossFile user group
     // (2) The supplemental context is from Supplemental Context but not UTG(unit test generator)
     const isUtg = await isTestFile(editor, DependencyGraphFactory.getDependencyGraph(editor.document.languageId))
-    const supplementalContexts: CodeWhispererSupplementalContext | undefined =
-        CodeWhispererUserGroupSettings.getUserGroup() === CodeWhispererConstants.UserGroup.CrossFile && !isUtg
```
Comment (Will-ShaoHua) [inline src/codewhisperer/util/editorContext.ts:98]: @leigaol the real logic using `usergroup` is here

Comment (Will-ShaoHua) [reply src/codewhisperer/util/editorContext.ts:98]: hence it's removed

File: src/codewhisperer/util/userGroupUtil.ts [hunk 3]
```diff
@@ -51,10 +51,7 @@ export class CodeWhispererUserGroupSettings {
...
-        const randomNum = Math.random()
-        const result = randomNum <= 1 / 2 ? UserGroup.Control : UserGroup.CrossFile
-
-        return result
```
Comment (leigaol) [inline src/codewhisperer/util/userGroupUtil.ts:57]: This change conflicts with https://github.com/aws/aws-toolkit-vscode/pull/3651/files#diff-46248c6fb980971704762e9444db5d26ea68a7326bd12ccbf92fea0ce755034eR55

File: src/codewhisperer/util/editorContext.ts (hunk 2 above)
Comment (leigaol) [reply src/codewhisperer/util/editorContext.ts:98]: OK. So regardless of the user group, cross file context is enabled. Correct?

File: src/codewhisperer/util/userGroupUtil.ts (hunk 3 above)
Comment (Will-ShaoHua) [reply src/codewhisperer/util/userGroupUtil.ts:57]: yea this one will be merged first, the UTG one should be the final look

Comment (Will-ShaoHua) [reply src/codewhisperer/util/userGroupUtil.ts:57]: Just seperate these 2 out in 2 PRs
1. remove cross file A/B
2. add usg A/B

File: src/codewhisperer/util/editorContext.ts (hunk 2 above)
Comment (Will-ShaoHua) [reply src/codewhisperer/util/editorContext.ts:98]: yes it's enabled regardless the usergorup

**Review by leigaol — APPROVED** [review]

**Pull request closed without merging**